data/mods/Magiclysm/worldgen/goblin_encampment.json

```diff
@@ -0,0 +1,578 @@
+        "         #..................................................................................................#           ",
+        "         #..................................................................................................#           ",
+        "         ####################################################################################################           ",
+        "                                                                                                                        ",
+        "                                                                                                                        ",
+        "                                                                                                                        ",
+        "                                                                                                                        "
+      ],
+      "palettes": [ "goblin_palette" ],
+      "items": {
+        "c": { "item": "stash_wood", "chance": 50 },
+        "h": { "item": "bed", "chance": 40, "repeat": [ 1, 2 ] },
+        "%": { "item": "trash", "chance": 40, "repeat": [ 1, 2 ] }
+      },
+      "place_nested": [
+        { "chunks": [ "goblin_center_nest" ], "x": 25, "y": 25 },
+        { "chunks": [ "goblin_center_nest" ], "x": 49, "y": 25 },
+        { "chunks": [ "goblin_center_nest" ], "x": 73, "y": 25 }
+      ],
+      "nested": { "1": { "chunks": [ [ "null", 20 ], [ "goblin_campfire", 80 ] ] } },
+      "place_monsters": [
+        { "monster": "GROUP_GOBLIN_STANDARD", "x": [ 24, 47 ], "y": [ 24, 46 ], "density": 0.1, "repeat": [ 1, 3 ] },
+        { "monster": "GROUP_GOBLIN_STANDARD", "x": [ 48, 71 ], "y": [ 24, 46 ], "density": 0.1, "repeat": [ 1, 3 ] },
+        { "monster": "GROUP_GOBLIN_STANDARD", "x": [ 72, 95 ], "y": [ 24, 46 ], "density": 0.1, "repeat": [ 1, 3 ] }
+      ]
+    }
+  },
+  {
+    "type": "palette",
+    "id": "goblin_palette",
+    "terrain": {
+      "3": "t_palisade_gate",
+      "l": "t_ladder_up",
+      "#": "t_wall_rammed_earth",
+      "t": "t_wall_wattle",
+      "W": "t_wall_wattle_half",
+      "w": "t_wall_wattle_broken",
+      " ": "t_region_groundcover_urban",
+      "T": "t_trunk",
+      "&": "t_palisade_pulley",
+      "|": "t_palisade",
+      ".": "t_dirtfloor",
+      "h": "t_dirtfloor",
+      "!": "t_dirtfloor",
+      "~": "t_dirtfloor",
+      "1": "t_dirtfloor",
+      "+": "t_door_makeshift_c",
+      "=": "t_water_murky"
+    },
+    "furniture": {
+      "h": "f_straw_bed",
+      "@": "f_logstool",
+      "c": "f_firering",
+      "Y": [ [ "f_boulder_small", 5 ], "f_boulder_medium", "f_boulder_large" ]
+    },
+    "items": {
+      "c": { "item": "goblin_cookpot", "chance": 50 },
+      "h": { "item": "bed", "chance": 40, "repeat": [ 1, 2 ] },
+      "!": { "item": "lair_loot_generic", "chance": 40, "repeat": [ 1, 2 ] },
```

Comment:
maybe swap to `lair_black_dragon`, this is still pretty low tier loot but with the possibility of some weapons and rare chances of magiclysm themed loot.